src/Collection/CollectionBuilder.php

```diff
@@ -139,7 +146,16 @@ public function addTask(TaskInterface $task)
         return $this;
     }
 
-    public function addCode(callable $code, $name)
+  /**
+   * Add arbitrary code to execute as a task.
+   *
+   * @see \Robo\Collection\CollectionInterface::addCode
+   *
+   * @param callable $code
+   * @param int|string $name
+   * @return $this
+   */
+    public function addCode(callable $code, $name = self::UNNAMEDTASK)
```

Comment:
Ah. Makes sense -- wasn't sure if it was kosher to use a FQSEN to that class here.